Source/WebKit/UIProcess/API/Cocoa/_WKAttachment.mm

```diff
@@ -109,6 +109,11 @@ - (NSString *)contentType
     return _utiType.get();
 }
 
+- (BOOL)preserveFidelity
```

Comment:
Nit: `preservesFidelity` might be a better name? It could be read like a verb...

Reply:
Went with `shouldPreserveFidelity`.